src/spikeinterface/core/tests/test_basesorting.py

```diff
@@ -102,6 +119,10 @@ def test_BaseSorting():
     for unit in sorting_clean.get_unit_ids():
         assert unit not in empty_units
 
+    sorting4 = sorting.to_numpy_sorting()
+    sorting5 = sorting.to_multiprocessing(n_jobs=2)
```

Comment:
can you add a test with `save` with more than ` job? This should also run on Windows

Reply:
yep

Reply:
NumpyFolder and NpzFolder do not have parralel writting.
The shared mem sorting is already tested in many place because we do `to_multiprocessing()` for waveform extractor